armi/reactor/converters/uniformMesh.py

```diff
@@ -188,11 +232,14 @@ def plotConvertedReactor(self):
             )
 
     def _setParamsToUpdate(self):
+        """Gather a list of parameters that will be mapped between reactors."""
         b = self._sourceReactor.core.getFirstBlock()
 
-        self.blockParamNames = b.p.paramDefs.inCategory("detailedAxialExpansion").names
+        self.blockParamNames = b.p.paramDefs.inCategory(
+            parameters.Category.detailedAxialExpansion
+        ).names
         self.reactorParamNames = self._sourceReactor.core.p.paramDefs.inCategory(
-            "neutronics"
+            parameters.Category.neutronics
```

Comment:
It seems odd to use a category dedicated to this purpose in the block params, but to reach for the oddly-specific "neutronics" category for the reactor params. This is nothing new, so shouldn't hold up the show, but maybe we should consider adding the neutronics reactor params to the detailedAxialExpansion category?

Reply:
good point. I'm thinking we should make the a `UniformMeshConverterForNeutronics` subclass in the neutronics package to really separate it. The conversion itself is generic, but the list of params that get translated could be different for different clients/subclasses.